.devcontainer/Dockerfile

```diff
@@ -0,0 +1,86 @@
+ARG DEBIAN=debian
+FROM debian:${DEBIAN}
+ARG PYVERSION=3
+
+# ---
+# Create the default user - vscode mounts workspace directory chowned to 1000:1000
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+RUN groupadd --gid $USER_GID $USERNAME \
+&& useradd --uid $USER_UID --gid $USER_GID -m $USERNAME \
+&& apt-get update \
```

Comment:
Why the mixture of `apt-get` here and `apt` below?

Reply:
No good reason (the bad one is "done at different times"). I'll change for consistency